ci-operator/step-registry/ovn/sdn-migration-rollback/ovn-sdn-migration-rollback-commands.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET=${TARGET:-OpenShiftSDN}
+
+oc patch MachineConfigPool master --type='merge' --patch '{"spec":{"paused":true}}'
+oc patch MachineConfigPool worker --type='merge' --patch '{"spec":{"paused":true}}'
+
+# Reset the spec.migration before we can set it to other value
+oc patch Network.operator.openshift.io cluster --type='merge' --patch '{"spec":{"migration":null}}'
+# Wait until CNO update the applied-cluster cm to the latest
+timeout 60s bash <<EOT
+until 
+  ! oc get network -o yaml | grep migration  > /dev/null
+do
+  echo "migration field is not cleaned by CNO"
+  sleep 3
+done
+EOT
+
+# Allow rollback
+# Change network to target network in Network.config.openshift.io the CR to trigger machine config update by MCO.
+oc patch Network.operator.openshift.io cluster --type='merge' --patch "{\"spec\":{\"migration\":{\"networkType\":\"${TARGET}\"}}}"
+oc patch Network.config.openshift.io cluster --type='merge' --patch "{\"spec\":{\"networkType\":\"${TARGET}\"}}"
+
+oc wait co network --for='condition=PROGRESSING=True' --timeout=30s
+# Wait until the multus pods are restarted
+timeout 300 oc rollout status ds/multus -n openshift-multus
+
+# Reboot all the nodes
```

Comment:
I'd prefer to use MCO for that, for instance we could add an MC with a new ssh key, this would make [MCO reboot all nodes sequentially](https://github.com/openshift/machine-config-operator/blob/master/docs/Update-SSHKeys.md), similar to what customers would do to avoid disruption

Reply:
That can not work with the SDN migration case. The MCO relies on a working cluster network to drain nodes, however, at this moment, neither ovn-kube nor openshift-sdn works.  So we have to reboot all the nodes at the same time so that the interface of each pod can be recreated.